spec/rubocop/cli/cli_options_spec.rb

```diff
@@ -730,7 +730,7 @@ def full_description_of_cop(cop)
                     ' ^',
                     'example2.rb:3:1: C: Inconsistent indentation ' \
                     'detected.',
-                    'def a',
+                    'def a...',
```

Comment:
It's [here](https://github.com/bbatsov/rubocop/blob/master/lib/rubocop/cop/mixin/autocorrect_alignment.rb#L42) that we highlight the whole node, in this case a `def` node. I wouldn't call it a bug. It's the easiest solution.